lib/ai/client.go

```diff
@@ -77,3 +77,22 @@ func (client *Client) Summary(ctx context.Context, message string) (string, erro
 
 	return resp.Choices[0].Message.Content, nil
 }
+
+// CommandSummary creates a command summary based on the command output.
+// The message history is also passed to the model in order to keep context
+// and extract relevant information from the output.
+func (client *Client) CommandSummary(ctx context.Context, messages []openai.ChatCompletionMessage) (string, error) {
```

Comment:
Doesn't this just perform a generic completion request? Shouldn't this function be responsible for injecting the summary prompt?

Reply:
Done :)